### PR TITLE
fix: H2->3 in <details> sections

### PR DESCRIPTION
**Pull request by jorgeorpinel**
so that they don't show up in the right ✋🏼 CONTENT sidebar; E.g.:

image

> Current https://dvc.org/doc/start/data-management/data-versioning

### CODE REVIEW

**Comment by jorgeorpinel** [comment]
Cc @iterative/websites is this the right approach, of should the engine ignore headers from `<details>` sections? Minor. Thanks

File: content/docs/user-guide/how-to/stop-tracking-data.md [hunk 1]
```diff
@@ -8,9 +8,7 @@ description: 'You can "un-track" files or directories added in error.'
 There are situations where you may want to "un-track" files or directories added
 in error to DVC.
 
-<details>
-
-## Expand to add a sample data `data.csv` file
+#Expand to add a sample data `data.csv` file
```
Comment (rogermparent) [inline content/docs/user-guide/how-to/stop-tracking-data.md:11]: Does removing the `<details>` here break something?

**Comment by rogermparent** [comment]
I'm honestly not sure what the proper approach would be for this, but if we want to prevent `details` blocks from showing up in the content nav that could be a totally viable feature- I don't think we deliberately set it to act either way, I think the `headers` data used for the content nav feature actually comes directly from `gatsby-plugin-remark` so we don't have great control over it currently.

File: content/docs/user-guide/how-to/stop-tracking-data.md (hunk 1 above)
Comment (jorgeorpinel) [reply content/docs/user-guide/how-to/stop-tracking-data.md:11]: Oh weird, this is not the change I intended ⏳

Comment (jorgeorpinel) [inline content/docs/user-guide/how-to/stop-tracking-data.md:11]:
```suggestion
<details>

### Expand to add a sample data `data.csv` file
```

**Review by rogermparent — APPROVED** [review]
Looks alright by me! I'll follow up on what we do with details headers during next websites meeting.

**Comment by jorgeorpinel** [comment]
It's nor a big deal, prob not worth addressing at this point. On the other hand I keep noticing https://github.com/iterative/gatsby-theme-iterative/issues/38 so may be something to check whether it's easy to fix.

**Pull request merged**